### ISSUE TITLE
Z3Optimizer not present in Windows compilation of Inox

### ISSUE THREAD

**Issue by MikaelMayer** [epfl-lara/inox#20]
I wanted to compile inox on windows by using  `sbt publish-local`. However, I get the following problem:

```
[info] Compiling 122 Scala sources to C:\...\inox\target\scala-2.11\classes...
[info] Wrote C:\...\inox\target\scala-2.11\inox_2.11-1.0.2-55-g499e6bc.pom
[info] Main Scala API documentation to C:\...\inox\target\scala-2.11\api...
[info] :: delivering :: ch.epfl.lara#inox_2.11;1.0.2-55-g499e6bc :: 1.0.2-55-g499e6bc :: integration :: Thu Apr 06 19:40:27 CEST 2017
[info]  delivering ivy file to C:\...\inox\target\scala-2.11\ivy-1.0.2-55-g499e6bc.xml
[error] C:\...\inox\src\main\scala\inox\solvers\z3\NativeZ3Optimizer.scala:8: object Z3Optimizer is not a member of package z3.scala
[error] import z3.scala.{Z3Optimizer => ScalaZ3Optimizer, _}
[error]        ^
```

I looked into the provided JAR file, but the Z3Optimizer class is indeed not present in the windows version of `unmanaged/scalaz3-win-64-2.11.jar`, whereas it appears correctly in the unix version of the same file.

What should I do to update this file?

**Comment by samarion** [comment]
Indeed, I had to make some updates in the ScalaZ3 API for MaxSMT so you'll have to build the latest ScalaZ3 for windows. I think you can go with the usual ScalaZ3 build procedure.

By the way, it would be nice if we could get the ScalaZ3 jar to work out of the box on Windows at some point instead of having to manually put the z3 lib files on the path...

**Comment by MikaelMayer** [comment]
Going to be fixed by #21
